src/main.cpp

```diff
@@ -37,7 +40,33 @@ int main(int argc, char** argv)
 #endif
     std::cerr << std::boolalpha;
 
+    // Setup sentry error handling.
+    sentry_options_t* options = sentry_options_new();
+    // TODO: Right now we just hardcode the key. We should probably get that from some kind of secret for release builds
+    sentry_options_set_dsn(options, "https://734f9ec9024f73e53701d59c3ffddfe3@o323038.ingest.sentry.io/4506257745510401");
+    // This is also the default-path. For further information and recommendations:
+    // https://docs.sentry.io/platforms/native/configuration/options/#database-path
+    std::string config_path = "";
+
+#if defined(__linux__)
+    config_path += getenv("HOME");
+    config_path += "/.local/share/cura/";
+#elif defined(__APPLE__) && defined(__MACH__)
+    config_path += getenv("HOME");
+    config_path += "/Library/Application Support/cura/";
+#elif defined(_WIN64)
+    config_path = "%APPDATA%\\cura\\";
+#endif
+    config_path += ".sentry-native";
+    sentry_options_set_database_path(options, config_path.c_str());
+    std::string version = "curaengine@";
+    version += std::string(CURA_ENGINE_VERSION);
```

Comment:
Why not use `fmt::format` here?
```suggestion
    std::string version = fmt::format("curaengine@"{}, CURA_ENGINE_VERSION);
```